sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/molecules/BottomSheetViewScreen.kt

```diff
@@ -0,0 +1,308 @@
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens.molecules
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import uk.gov.hmrc.components.compose.atom.button.PrimaryButton
+import uk.gov.hmrc.components.compose.molecule.bottomsheet.BottomSheetView
+import uk.gov.hmrc.components.compose.molecule.selectrow.SelectRowView
+import uk.gov.hmrc.components.compose.molecule.titleBody.H5TitleBodyView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ExamplesSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ScreenScrollViewColumn
+
+@Composable
+fun BottomSheetViewScreen() {
```

Comment:
Your approach is perfectly fine, but I don't know why in this scenario it's somehow difficult for me, Could you please take a look at this screenshot if it clarifies things?
![Screenshot 2024-02-28 at 12 29 55](https://github.com/hmrc/android-components/assets/11981999/e32b1d9b-61ce-471a-99b2-36895b29f2b0)

Reply:
I think it's good to group all the state at the top of the composable so it's easy to see whats there, but I've added comments and grouped the examples together more so hopefully that's easier to read?